src/get-started/data-proofs.md

```diff
@@ -150,35 +147,66 @@ impl fabric::ServiceFactory for ServiceFactory {
 Similarly to a simple cryptocurrency demo we need to declare the data that we
 will store in our blockchain, i.e. the `Wallet` type:
 
+```protobuf
+message Wallet {
+  exonum.PublicKey pub_key = 1;
+  string name = 2;
+  uint64 balance = 3;
+  uint64 history_len = 4;
+  exonum.Hash history_hash = 5;
+}
+```
+
 ```rust
-encoding_struct! {
-    struct Wallet {
-        pub_key:            &PublicKey,
-        name:               &str,
-        balance:            u64,
-        history_len:        u64,
-        history_hash:       &Hash,
-    }
+use exonum_derive::ProtobufConvert;
+
+#[derive(Clone, Debug, ProtobufConvert)]
+#[exonum(pb = "proto::Wallet", serde_pb_convert)]
+pub struct Wallet {
+    pub pub_key: PublicKey,
+    pub name: String,
+    pub balance: u64,
+    pub history_len: u64,
+    pub history_hash: Hash,
```

Comment:
Shouldn't we clearly specify that Wallet struct must be placed into the wallet.rs file?

Reply:
Added this mention to all files.